go.mod

```diff
@@ -0,0 +1,51 @@
+module github.com/paketo-community/builder-ubi-base
+
+go 1.18
```

Comment:
We should probably start off with `go 1.20` given that `go 1.18` is likely to go out of support shortly.

Reply:
will update and test.

Reply:
done